mlflow/models/rag_signatures.py

```diff
@@ -0,0 +1,56 @@
+from dataclasses import dataclass, field
+from typing import List, Optional, Union
+
+
+@dataclass
+class Message:
+    role: str = "user"  # "system", "user", or "assistant"
+    content: str = "What is mlflow?"
```

Comment:
Wondering why we need default values here

Reply:
Today its not needed, but in the future, we can use the dataclass defaults as the input example.